### PR TITLE
add support for logs

### PR DESCRIPTION
**Pull request by hassenius**

### CODE REVIEW

**Comment by hassenius** [comment]
cleaned up code a bit from last pr

File: README.md [hunk 1]
```diff
@@ -22,6 +23,24 @@ lm.emit_metric(name='logmet.test.2', value=2)
 lm.emit_metric(name='logmet.test.3', value=3)
 ```
 
+## Sending logs
+```python
+import logmet
+
+lm = logmet.Logmet(
+    logmet_host='logs.opvis.bluemix.net',
+    logmet_port=9091,
```
Comment (mrodden) [inline README.md:32]: I think I will deprecate these keywords in favor of logs_host, logs_port, and metrics_host, metrics_port. That way a user can use the same Logmet object to send either or both.

File: logmet/__init__.py [hunk 2]
```diff
@@ -150,6 +172,105 @@ def wrap_for_send(messages):
...
+            parts.extend([
+                self._pack_int(len(key)),
+                key,
+                self._pack_int(len(value)),
+                value,
+            ])
+
+        return self._pack_int(total_keys) + ''.join(parts)
+
+    def _encode_unicode(self, obj):
```
Comment (mrodden) [inline logmet/__init__.py:217]: This functions code is in a couple places, and its really more a "to_bytes" method, but I will take care of that after merge.

**Review by mrodden — COMMENTED** [review]
@hassenius are you OK with me changing the Logmet constructor to explicit keywords on the log vs metrics host? It will probably affect your code that depends on the module.

**Comment by hassenius** [comment]
@locke105 I'm fine with moving to target specific keys. Won't really affect me as I've been using it in openwhisk where I've needed to copy/paste the code directly into the action file.

**Comment by mrodden** [comment]
This should be merged with commit ed2b78e471ce2c7cc619c919a281e019421f4ec6

**Pull request closed without merging**